frontend-nannyway/src/components/header/AppBarSecondaryColor.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Grid from "@material-ui/core/Grid";
+import NavMenuSecondaryColor from "./NavMenuSecondaryColor";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
```

Comment:
Not sure if you actually need this, since the parent is not a `flexbox`, sorry if I missed that earlier

Reply:
Right. Done ;)